pkg/prometheus/server/operator.go

```diff
@@ -1125,12 +1126,15 @@ func (c *Operator) createOrUpdateConfigurationSecret(ctx context.Context, p *mon
 	}
 
 	if p.Spec.Alerting != nil {
-		for i, am := range p.Spec.Alerting.Alertmanagers {
-			if err := prompkg.ValidateAlertmanagerEndpoints(am, p); err != nil {
+		ams := p.Spec.Alerting.Alertmanagers
+
+		for i, am := range ams {
```

Comment:
Can't recall why I brought this loop (calling `validateAlertmanagerEndpoints()`) outside of `addAlertmanagerEndpointsToStore()`, should I just move it back?